relocation/java5/pom.xml

```diff
@@ -35,7 +36,7 @@
     <name>Relocation (core, java-5)</name>
 
     <properties>
-        <root.basedir>${project.parent.basedir}/..</root.basedir>
+        <root.basedir>${project.parent.basedir}</root.basedir>
```

Comment:
What is this property used for? I am having issues findings anything about it in the Maven documentation.

Reply:
That is the basedir from the root module.
It is not a standard Maven property, but one I use to locate configuration files relative to the root directory (e.g. the header file for the license plugin)